package.json

```diff
@@ -17,42 +17,33 @@
     "@18f/stylelint-rules": "^2.0.0",
     "axios": "^0.18.0",
     "axios-mock-adapter": "^1.15.0",
-    "babel-core": "^6.26.3",
-    "babel-jest": "^23.0.1",
```

Comment:
Definitely need these two!

Reply:
Yarn figures out that we need it – do we need to put it as a top-level dependency?

Reply:
(i.e., `jest` pulls it in, so does  `babel-loader`)

Reply:
```
➜  e-QIP-prototype git:(remove-unused) yarn
yarn install v1.7.0
[1/5] 🔍  Resolving packages...
[2/5] 🚚  Fetching packages...
[3/5] 🔗  Linking dependencies...
warning " > babel-loader@7.1.4" has unmet peer dependency "babel-core@6".
...
```

Reply:
Good to go after that's re-added!